curvefs/src/client/dir_buffer.h

```diff
@@ -35,18 +38,28 @@ struct DirBufferHead {
     bool wasRead;
     size_t size;
     char *p;
+    DirBufferHead()
+        : wasRead(false),
+          size(0),
+          p(nullptr) {}
 };
 
 class DirBuffer {
  public:
-    DirBuffer() {}
+    DirBuffer() :
```

Comment:
directory buffer?

Reply:
done

Reply:
why do we need directory buffer

Reply:
we need a buffer to return to fuse， if use temporary buffer,  to build this buffer will list all dentry in dir , it will cost lots of time.